lib/ansible/modules/remote_management/hpilo/hpilo_boot.py

```diff
@@ -147,8 +154,9 @@ def main():
     image = module.params['image']
     state = module.params['state']
     force = module.params['force']
+    ssl_version = getattr(hpilo.ssl, 'PROTOCOL_' + module.params.get('ssl_version').upper().replace('V','v'))
```

Comment:
The `upper()` and `replace()` calls here seem unneeded given that the input is very strictly formatted already.
But that's not a showstopper either.